blocks/cards/cards.js

```diff
@@ -46,6 +46,11 @@ export default function decorate(block) {
       [...li.children].forEach(addCardChildrenClasses);
     }
 
+    if (a && block.classList.contains('video-modal') && document.querySelector('.modal-fragment') !== null) {
```

Comment:
Would the page author explicitly need to specify `video-modal` or could we automatically detect this if the block has an `a` element with `href` containing `/fragments/modals/` ? If that's possible, we can reduce a bit of authoring overhead..

Reply:
yes, you are right, if the authors has used URLs from "fragments/modals" folder, then it is implicit that it will open as a Modal. Let me revisit this check.